<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRuleta</class>
 <widget class="QDialog" name="frmRuleta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ruleta de protecci√≥n</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QGraphicsView" name="gvRuleta">
      <property name="cacheMode">
       <set>QGraphicsView::CacheBackground</set>
      </property>
     </widget>
     <widget class="QFrame" name="frameInfo">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_info_ico">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/gr-lida.qrc">:/img16/informacion.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lb_info_titulo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="btnCerrarInfo">
         <property name="icon">
          <iconset resource="../res/gr-lida.qrc">
           <normaloff>:/images/tab_close.png</normaloff>:/images/tab_close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>9</width>
           <height>9</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextBrowser" name="txtInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QToolButton" name="actionSalir">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/cerrar.png</normaloff>:/img16/cerrar.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>16</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="actionInfo">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/informacion.png</normaloff>:/img16/informacion.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>16</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="actionZoomIn">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/zoom_in.png</normaloff>:/img16/zoom_in.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="actionZoomOut">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/zoom_out.png</normaloff>:/img16/zoom_out.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="actionSizeNormal">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/zoom.png</normaloff>:/img16/zoom.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="ruleta_Cartas">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_carta_top">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Carta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxCartaTop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_carta_bot">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>sobre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxCartaBot">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ruleta_D">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_D">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ruleta D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_D">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ruleta_C">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_C">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ruleta C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_C">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ruleta_B">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_B">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ruleta B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_B">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ruleta_A">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_A">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ruleta A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_A">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gvRuleta</tabstop>
  <tabstop>cbxCartaTop</tabstop>
  <tabstop>cbxCartaBot</tabstop>
  <tabstop>slider_D</tabstop>
  <tabstop>slider_C</tabstop>
  <tabstop>slider_B</tabstop>
  <tabstop>slider_A</tabstop>
  <tabstop>actionInfo</tabstop>
  <tabstop>actionZoomIn</tabstop>
  <tabstop>actionZoomOut</tabstop>
  <tabstop>actionSizeNormal</tabstop>
  <tabstop>txtInfo</tabstop>
  <tabstop>btnCerrarInfo</tabstop>
  <tabstop>actionSalir</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
